plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -440,7 +453,11 @@ private function get_authorization_id_for_capture( $order ) {
 					$order->update_meta_data( '_paypal_status', WC_Gateway_Paypal_Constants::STATUS_AUTHORIZED );
 					$order->save();
 				} else {
+					// Store '_paypal_authorization_checked' flag to prevent repeated API calls for orders with no authorization data.
 					WC_Gateway_Paypal::log( 'Authorization ID not found in PayPal order details. Order ID: ' . $order->get_id() );
+					$order->update_meta_data( '_paypal_authorization_id', '' );
```

Comment:
Nit: would it not be better to delete the value rather than storing `''`?
```suggestion
					$order->delete_meta_data( '_paypal_authorization_id' );
```

Reply:
I have removed this line completely. At this point, the order is not supposed to have the `_paypal_authorization_id` meta. So I decided neither to delete nor set it empty.